scripts/bulk_set_requester_pays.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+
+if (( $# != 1 )); then
+  echo "Usage: $0 /path/to/inputfile.csv"
+  echo "inputfile.csv should contain the following column headers in order: project, bucket, toggle_type"
+  echo 'The Terra bucket paths can be formatted as "gs://fc-XXXXX" or "fc-XXXXX"'
+  echo "The toggle_type options are on|off for enabling and disabling requester pays"
+  echo "NOTE: this script requires you to be authed as your firecloud.org admin account."
+  exit 0
+fi
+
+
+CSVIN=$1
+USER_EMAIL=$(gcloud config get-value account)
+SERVICE_ACCOUNT="requester-pays@broad-dsde-prod.iam.gserviceaccount.com"
+MEMBER="serviceAccount:${SERVICE_ACCOUNT}"
+SLEEP_SEC=15
+
+ROLE="organizations/386193000800/roles/RequesterPaysToggler"
+
+
+# 1. confirm csv isn't missing required values
+
+echo "Validating input file..."
+
+# check csv headers
+if [[ $(head -1 $CSVIN) != $'project,bucket,toggle_type\r' ]]; then
+  echo "Missing or incorrect column headers"
+  echo "inputfile.csv should contain the following column headers in order: project, bucket, toggle_type"
+  exit 0
+fi
+
+# check for any missing values
+missing=false
+while IFS="," read -r project bucket toggle_type; do
+  if [[ $project == "" ]]
+  then
+    missing=true
+  elif [[ $bucket == "" ]]
+  then
+    missing=true
+  elif [[ $toggle_type == "" ]]
+  then
+    missing=true
+  fi
+done < $CSVIN
+if ( $missing ) ; then
+  echo "ERROR: Missing values in input CSV file."
+  echo "Input CSV file should contain the following columns and corresponding values: project, bucket, toggle_type"
+  exit 0
+else
+  echo "Valid input file."
+fi
+
+# 2. grant the "RequesterPaysToggler" role to the service account for all projects in the list
+# you need to be logged in with your firecloud account
+echo "Enabling permissions for ${SERVICE_ACCOUNT} to set Requester Pays"
+while IFS="," read -r PROJECT_ID; do
+  if gcloud projects add-iam-policy-binding $PROJECT_ID --member=$MEMBER --role=$ROLE --no-user-output-enabled; then
+    echo "Access granted to $PROJECT_ID"
+  fi
```

Comment:
do we want an else block here to let the user know if granting access failed for any of the projects?

Reply:
added in [5a85d9f](https://github.com/broadinstitute/horsefish/pull/148/commits/5a85d9ff79700393211b479341fce6b1020098a5)